src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,24 @@
+.clock {
+  margin: 25vh auto;
```

Comment:
As you can see in the [link](https://snipboard.io/0Ny4jH.jpg) the clock isn't in the center and there is a scroll in the page. You can solve this problem by giving display flex to parent and setting height to viewport height (for the parent.)